src/components/charts/BarChart.tsx

```diff
@@ -125,28 +123,21 @@ const BarChart = () => {
     setSessionData(data || []);
   };
 
-  // const getHardCodedSessionData = () => {
-  //   setIsLoading(false);
-  //   setSessionData(sessionsJSON);
-  // };
-
   const changeStartTime = (value: string) => {
     const formattedTime = formatTime(value);
     setStartTime(value);
     let defaultDate: string = "";
-
-    defaultDate = !startDate
-      ? `${new Date().toISOString().split("T")[0]}`
-      : formatDate(startDate, DATE_FORMAT_2);
-
+    defaultDate = startDate
+      ? formatDate(startDate, DATE_FORMAT_2)
+      : `${new Date().toISOString().split("T")[0]}`;
     setStartDateTime(`${defaultDate}T${formattedTime}`);
   };
 
   const changeStartDate = (value: string) => {
     setStartDate(value);
     const selectedDate = new Date(value);
     let defaultTime: string = "";
-    defaultTime = !startTime ? formatTime(new Date()) : formatTime(startTime);
+    defaultTime = startTime ? formatTime(startTime) : formatTime(new Date());
```

Comment:
Just a suggestion for less verbosity.
` defaultTime = formatTime(startTime ? startTime : new Date())`

Reply:
done